OpenSourcePolitics/decidim-module-friendly_signup
# Fix/email format

🎩 Description
On the signup form, a user can create an XSS injection through the email field. 

📌 Related Issues
[Notion card](https://www.notion.so/opensourcepolitics/CEA-Reflected-XSS-on-friendly-signup-confirmation-code-b14756bd96ed4b1291780e55f26d2bf0?pvs=4)

Testing

FRONT

1. Access the signup form.
2. Fill in the fields, and for the email field, enter 
- `"<script>alert('Jakick')</script>"@gmail.com` or 
- `"<script>alert('XSS')</script>"@example.org` or 
- an email with ' " > or >.
4. Submit the form.
5. Verify that there is no JavaScript alert window and that the form submission is prevented.
